app/scripts/lib/constants.js

```diff
@@ -112,8 +112,13 @@ define(function (require, exports, module) {
     // Login delay for iOS broker
     IOS_V1_LOGIN_MESSAGE_DELAY_MS: 10000,
 
-    UNBLOCK_CODE_LENGTH: 8
+    UNBLOCK_CODE_LENGTH: 8,
+
+    MARKETING_ID_SPRING_2015: 'spring-2015-android-ios-sync',
+    MARKETING_ID_AUTUMN_2016: 'autumn-2016-connect-another-device',
+
+    DOWNLOAD_LINK_ANDROID: 'https://app.adjust.com/2uo1qc?campaign=%(campaign)s&creative=%(creative)s&adgroup=android',
```

Comment:
Add a `_TEMPLATE` suffix to the download links so future devs know they need to do some interpolation.